README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
>  to make easy-to-deploy static linked or minimally dynamic linked programs

Programs, not libraries. `musl-gcc -static -Os hello.c` in example `musl-gcc` used to compile final executable.

As a general though if `musl-gcc` is just a wrapper for `gcc` then why we need to download compiler at all? It will be easier to add some flags to standard `gcc`. Though I'm not sure about all the details.